.github/workflows/build_docker.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       - uses: actions/setup-node@v3
         with:
           node-version: ${{ env.NODE_VERSION }}
-          registry-url: https://npm.pkg.github.com/
+          # registry-url: https://npm.pkg.github.com/
```

Comment:
I wonder if other ref to this registry should be removed
![image](https://user-images.githubusercontent.com/10514677/218299407-1cb9510b-cdf7-4134-aa40-7574bf8f9ba9.png)

also maybe we should remove completely instead of commenting, wdyt?

Reply:
Done